protocols/request-response/Cargo.toml

```
@@ -16,12 +16,20 @@ futures = "0.3.28"
 instant = "0.1.11"
 libp2p-core = { workspace = true }
 libp2p-swarm = { workspace = true }
+libp2p-swarm-derive = { workspace = true, optional = true }
```

I think there was another thread somewhere but I can't find it now.

Does this work? (incl the next 3 suggestions)
```suggestion
libp2p-swarm = { workspace = true }
```

---

Yes, this works well. Thanks!